src/main/webapp/resources/js/pages/projects/samples-metadata-import/components/SampleMetadataImportWizard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { PageHeader, Space } from "antd";
+import { SampleMetadataImportSteps } from "./SampleMetadataImportSteps";
+import { setBaseUrl } from "../../../../utilities/url-utilities";
+
+/**
+ * React component that displays the Sample Metadata Uploader Wizard wrapper.
+ * @prop {number} currentStep - the current step, starting with zero
+ * @prop {string} currentStatus - the status of the current step
+ * @prop {any} children - the status of the current step
+ * @returns {*}
+ * @constructor
+ */
+export function SampleMetadataImportWizard({
+  currentStep,
+  currentStatus,
+  children,
+}) {
+  const { projectId } = useParams();
+
+  return (
+    <Space direction="vertical" size="large" style={{ width: `100%` }}>
```

Comment:
`size="large"` is actually not an option only small, I think large is the default.

Reply:
![image](https://user-images.githubusercontent.com/325703/140802433-6e9b5480-5d5c-4e3d-af64-5e36b544b24f.png)

The ant design documentation says 'small' is the default and 'large' is an option.

Reply:
:facepalm: